mypy.ini

```diff
@@ -341,6 +341,19 @@ no_implicit_optional = true
 warn_return_any = true
 warn_unreachable = true
 
+[mypy-homeassistant.components.onewire.*]
+check_untyped_defs = true
+disallow_incomplete_defs = true
+disallow_subclassing_any = true
+disallow_untyped_calls = true
+disallow_untyped_decorators = true
+disallow_untyped_defs = true
+no_implicit_optional = true
+strict_equality = true
```

Comment:
Are you adding this manually? It doesn't match sections nearby, please run `hassfest` again.

Reply:
I had issues with hassfest so I edited it manually. Now I've re-run hassfest again.